server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -1655,7 +1655,7 @@ public void loadNewTsFile(TsFileResource newTsFileResource)
    */
   private int findInsertionPosition(TsFileResource newTsFileResource) {
     File tsfileToBeInserted = newTsFileResource.getFile();
-    long newFilePartitionId = Long.parseLong(tsfileToBeInserted.getParent());
+    long newFilePartitionId = Long.parseLong(tsfileToBeInserted.getParentFile().getName());
```

Comment:
The parent file of the file may not be its time range. It's better to get time range from the time.
Such as:
`load /data/1584445970309-101-1.tsfile`

Reply:
Fixed. But there is still one thing remaining, what if the file belongs to more than one partition? Currently, I will report an error and a delicate solution will be left as a future job.

Reply:
Yes, I agree with you.